docs/develop/develop-add-ins-visual-studio.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Develop Office Add-ins with Visual Studio
+description: How to develop Office Add-ins with Visual Studio
+ms.date: 12/28/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins with Visual Studio
+
+This article describes how to use Visual Studio to develop an Office Add-in. If you've already created your add-in, you can skip ahead to the [Develop the add-in using Visual Studio](#develop-the-add-in-using-visual-studio) section.
+
+> [!NOTE]
```

Comment:
Is it worth adding that Yeoman is a better choice if you are more familiar with Node.js as well

Reply:
It's probably okay to omit this info here, since the other article that's linked to from this note contains info that says Yeoman creates a Node.js project.